src/debugpy/common/__init__.py

```diff
@@ -4,5 +4,13 @@
 
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import os
+
 
 __all__ = []
+
+# The lower time bound for assuming that the process hasn't spawned successfully.
```

Comment:
Should we mention the units (seconds) for this? or is this common enough that people will know.

Reply:
All time spans are customarily in seconds in Python, so I think the comment is descriptive enough. I'm hoping that new defaults are "good enough" for all practical uses, so users wouldn't really need to be aware of those environment variables - I added them there mostly to provide a workaround if it turns out to be wrong, but I don't think we should document this or treat this as a stable API.